myria/relation.py

```diff
@@ -0,0 +1,71 @@
+from dateutil.parser import parse
+from itertools import izip
+from myria import MyriaConnection, MyriaError
+from schema import MyriaSchema
+
+class MyriaRelation(object):
+  DefaultConnection = MyriaConnection(hostname='localhost', port=8753)
+
+  # MyriaRelation({'userName': 'public', 'programName': 'adhoc', 'relationName': 'relation'})
+  # or
+  # MyriaRelation('public:adhoc:relation')
+  # or
+  # MyriaRelation('relation') # Defaults are 'public' and 'adhoc'
+  def __init__(self, relation, connection=DefaultConnection, *args, **kwargs):
```

Comment:
The variadic stuff is a relic of my internal use cases -- good catch!

Slight tangent related to the `kwargs['schema']` hidden parameter: 

In it's current form `MyriaRelation` conflates new relations (schema _must_ be specified on instantiation) and existing relations (schema cannot be specified -- it should come from the API).  

I haven't settled on a structure that I'm happy with.  `MyriaRelation.newRelation('foo')` seems awkward, as does `NewMyriaRelation('foo')`.  Thoughts?